### ISSUE TITLE
DNS-Resolver inside Libreoffice Docker Container Installation

### ISSUE THREAD

**Issue by digitalshow** [bigbluebutton/bbb-install#385]
The package `bbb-libreoffice` tries to build a docker container and add some debian packages. For me, this failed during the installation of the package:

```
Package locales-all is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'locales-all' has no installation candidate
E: Unable to locate package libxt6
E: Unable to locate package libxrender1
The command '/bin/sh -c apt -y install locales-all fontconfig libxt6 libxrender1' returned a non-zero code: 100
dpkg: error processing package bbb-libreoffice-docker (--configure):
 installed bbb-libreoffice-docker package post-installation script subprocess returned error exit status 100
```

I found out this was a name resolution issue, apt update failed to fetch the package lists (I still wonder why the Dockerfile didn't fail at that point already). The recommended Ubuntu Server 18.04 uses systemd-resolved, which presents a local caching resolver and registers this at `/etc/resolv.conf`. Docker seems to notice that that one is not reachable inside containers so it falls back to Google's DNS servers and registers them at the container's `/etc/resolv.conf`. Inside our network, these are not reachable. My solution was to use our local DNS resolver directly inside the host's `/etc/resolv.conf`, which will be used by docker:
```
rm /etc/resolv.conf
echo "nameserver 10.11.12.13" > /etc/resolv.conf
```

Do we want to add that to the Documentation? If not, this issue might still help people hitting the same problem.

**Comment by ffdixon** [comment]
Added to docs -- thanks.

  https://docs.bigbluebutton.org/2.3/install.html#package-locales-all-is-not-available

**Comment by poorya-abbasi** [comment]
Greetings,
I still can't manage to get this working. What DNS server should I use inside my /etc/resolv.conf?

**Comment by arnounesco** [comment]
this can no longer be correct as the package bbb-libreoffice does not exis, there is only a package bbb-libreoffice-docker

trying this solution did not solve the problem on Ubuntu 18.
if you do 
```
systemd-resolve --status
```
after doing 
```
echo "nameserver 8.8.8.8" > /etc/resolv.conf
```
 you will not see the new nameserver there, anyhow rebooting the machine will overwrite you settings again

**Comment by digitalshow** [comment]
> this can no longer be correct as the package bbb-libreoffice does not exis, there is only a package bbb-libreoffice-docker

I'm sorry, the correct package name is and was `bbb-libreoffice-docker` as seen in the error message, I changed it above.


> you will not see the new nameserver there, anyhow rebooting the machine will overwrite you settings again

That happens because in a normal systemd-resolved setup `/etc/resolv.conf` is a symlink to a file managed by systemd-resolved. That's why I started with `rm /etc/resolv.conf` removing the symlink. I just noticed that that step was not taken to the documentation. I'm opening a pull request for that.
